### PR TITLE
Add cutomization error for MaxUploadSize

### PR DESCRIPTION
**Pull request by kskumgk63**
Hi, I want to customize error even if using `MaxUploadSize`, so I added this file.
And I fixed an error when I stated the example app.
This PullRequest is related to https://github.com/99designs/gqlgen/issues/1089#issue-576091601.

I have:
 - [ ] Added tests covering the bug / feature (see [testing](https://github.com/99designs/gqlgen/blob/master/TESTING.md))
 - [ ] Updated any relevant documentation (see [docs](https://github.com/99designs/gqlgen/tree/master/docs/content))

### CODE REVIEW

File: example/fileupload/server/server.go [hunk 1]
```diff
@@ -7,28 +7,18 @@ import (
...
 
 func main() {
 	http.Handle("/", playground.Handler("File Upload Demo", "/query"))
 	resolver := getResolver()
 
-	var mb int64 = 1 << 20
-
-	srv := handler.New(fileupload.NewExecutableSchema(fileupload.Config{Resolvers: resolver}))
-
-	srv.AddTransport(transport.MultipartForm{
```
Comment (vektah) [inline example/fileupload/server/server.go:28]: this example is supposed to show how to configure the transport directly, not use the defaults

File: example/fileupload/server/server.go [hunk 2]
```diff
@@ -55,46 +46,52 @@ func getResolver() *fileupload.Stub {
...
 				return []*model.File{}, err
 			}
 			resp = append(resp, &model.File{
-				ID:      i + 1,
-				Name:    req[i].File.Filename,
-				Content: string(content),
+				ID:          i + 1,
+				Name:        req.File.Filename,
+				Content:     string(content),
+				ContentType: req.File.ContentType,
```
Comment (vektah) [inline example/fileupload/server/server.go:88]: these changes seem unrelated?

File: example/fileupload/server/server.go (hunk 1 above)
Comment (kskumgk63) [reply example/fileupload/server/server.go:28]: I could not start example app on `master` without this changes.

What I did
1. `git checkout master`
2. go run

```
❯ go run example/fileupload/server/server.go 
2020/04/22 13:20:32 connect to http://localhost:8087/ for GraphQL playground
```

3. checking this error on developer tool on Google Chrome.

スクリーンショット 2020-04-22 13 21 09

> this example is supposed to show how to configure the transport directly, not use the defaults

could you check this [commit](https://github.com/99designs/gqlgen/pull/1093/commits/0056b5d9d87a3fab26f84b7612b9e74c3ecd9859)?

File: example/fileupload/server/server.go (hunk 2 above)
Comment (kskumgk63) [reply example/fileupload/server/server.go:96]: According to https://github.com/99designs/gqlgen/pull/1074 changes, I think it is related.

**Comment by vektah** [comment]
Closing this as I don't think error messages need to be customized. If I ship this one, then theres another 100 prs full of similar requests.

If you think the default isn't good, propose an update
If you really do need to change it for your own reasons, create a copy of this transport and change it.
